Add D3D12On7 (#132)
terrafx/terrafx.interop.windows: generation/other/d3d11on12on7/header.txt

// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.

// Ported from d3d11on12on7.h in the https://www.nuget.org/packages/Microsoft.Direct3D.D3D12On7 nuget package, version 1.1.0

> // Original source is Copyright © Microsoft. All rights reserved. Licensed under the University of Illinois Open Source License.

Reviewer: Perhaps:
```suggestion
// Original source is Copyright © Microsoft. All rights reserved. License details can be found here: https://www.nuget.org/packages/Microsoft.Direct3D.D3D12On7/1.1.0/License
```
